keyboards/mmkeyboard/class60/soldered/info.json

```diff
@@ -0,0 +1,178 @@
+{
+    "manufacturer": "MMkeyboard",
+    "keyboard_name": "Class60",
+    "maintainer": "lexbrugman",
+    "bootloader": "uf2boot",
+    "build": {
+        "lto": true
+    },
+    "development_board": "bluepill",
```

Comment:
looking at [Geekhack thread](https://geekhack.org/index.php?topic=120039.0) for this board, there's no mention of a [Bluepill development board](https://stm32-base.org/boards/STM32F103C8T6-Blue-Pill). 
Could `"processor"` and `"bootloader"` for this board be specified, replacing `"development_board"`?